src/app/index.route.js

```diff
@@ -15,4 +16,11 @@
     });
   }
 
+  /** @ngInject */
+  function runBlock($rootScope) {
+    $rootScope.$on('$stateChangeSuccess', function(e, toState) {
+      $rootScope.ACTIVE_STATE = toState;
```

Comment:
Answering for both ("why $rootScope?"), they were meant to be used in the view.

`USER_ID` didn't end up being used in a view, so it can be refactored to just return from an accessor method (`firebaseDriver.getUserId()`).

For `ACTIVE_STATE`, it's used here: https://github.com/stephenplusplus/phoenix/blob/e56f934ab3e54d058353f3a8565bc2e2c2f648f5/src/app/projects/projects.html#L3